flycheck.el

```diff
@@ -7744,7 +7744,12 @@ Otherwise return the previously used cache directory."
 Look up the directory hierarchy from FILE for a directory
 containing a file that matches REGEXP."
   (locate-dominating-file
-   file
+   ;; We must not pass regular file here because
+   ;; `locate-dominating-file' will just pass it into it's second
+   ;; lambdaargument . But our lambda expects only directories.
```

Comment:
"lambdaargument" -> "lambda argument", and there's an extra space before the dot "."